appium/webdriver/extensions/screen_record.py

```diff
@@ -73,9 +73,7 @@ def start_recording_screen(self, **options):
                 but doing so results in larger movie files.
 
         Returns:
-            bytes: Base-64 encoded content of the recorded media file or an empty string
-                if the file has been successfully uploaded to a remote location
-                (depends on the actual `remotePath` value).
+            None
```

Comment:
`None` comes from http://appium.io/docs/en/commands/device/recording-screen/start-recording-screen/ .
However if `stop_screen_recording` isn't called after previous `start_screen_recording`, `bytes` will return.

https://github.com/appium/appium-android-driver/blob/1e019d6865db66d0ebcd70e4b198f0de39c3e632/lib/commands/recordscreen.js#L235

Sorry for missing my checks, I'll fix here.